core/services.go

```diff
@@ -214,6 +214,18 @@ func (app *Application) updateMessage(user *model.CoreToken, message *model.Mess
 	return nil, fmt.Errorf("missing id or record")
 }
 
+func (app *Application) updateReadMessage(orgID string, appID string, ID string, userID *string) (*model.Message, error) {
+	message, err := app.storage.GetMessage(orgID, appID, ID)
+	if err != nil {
+		return nil, nil
+	}
```

Comment:
It doesn't give any value in the context of this feature. app.storage.GetMessage is not needed at all.